x-pack/dockerlogbeat/handlers.go

```diff
@@ -26,6 +28,26 @@ type StopLoggingRequest struct {
 	File string
 }
 
+// CapabilitiesResponse represents the response to a capabilities request
+type CapabilitiesResponse struct {
+	Err string
+	Cap logger.Capability
+}
+
+// LogsRequest represents the request object we get from a `docker logs` call
+type LogsRequest struct {
```

Comment:
I don't see this type being used outside this package. Let's keep it private to this package.